kratos/tests/cpp_tests/strategies/strategies/test_strategies.cpp

```diff
@@ -279,6 +282,67 @@ namespace Kratos
             }
         }
 
+        /**
+         * Checks if the Newton Rapshon strategy performs correctly the resolution of the system
+         */
+
+        KRATOS_TEST_CASE_IN_SUITE(DisplacementArcLengthStrategy, KratosCoreFastSuite)
+        {
+            Model current_model;
+
+            constexpr double tolerance = 1e-6;
+
+            ModelPart& model_part = current_model.CreateModelPart("Main");
+
+            SchemeType::Pointer pscheme = SchemeType::Pointer( new ResidualBasedIncrementalUpdateStaticSchemeType() );
+            LinearSolverType::Pointer psolver = LinearSolverType::Pointer( new SkylineLUFactorizationSolverType() );
+            ConvergenceCriteriaType::Pointer pcriteria = ConvergenceCriteriaType::Pointer( new ResidualCriteriaType(1.0e-4, 1.0e-9) );
+            BuilderAndSolverType::Pointer pbuildandsolve = BuilderAndSolverType::Pointer( new ResidualBasedBlockBuilderAndSolverType(psolver) );
+            Parameters settings(R"({
+
+                "max_iteration"            : 30,
+                "compute_reactions"        : true,
+                "reform_dofs_at_each_step" : true,
+                "move_mesh_flag"           : true,
+                "advanced_settings"        : {
+                    "desired_iterations"       : 4,
+                    "max_radius_factor"        : 10.0,
+                    "min_radius_factor"        : 0.1,
+                    "loads_sub_model_part_list": [],
+                    "loads_variable_list"      : []
+                }
```

Comment:
```suggestion
                "desired_iterations"       : 4,
                "max_radius_factor"        : 10.0,
                "min_radius_factor"        : 0.1,
                "loads_sub_model_part_list": [],
                "loads_variable_list"      : []
```

from the strategy POV those are not advanced settings